specs/app_settings/01_main.rst

```diff
@@ -0,0 +1,225 @@
+==========================================================
+App settings
+==========================================================
+
+=====  ============  =========  ==========
+epic      title       author     created
+=====  ============  =========  ==========
+`82`_  App settings  tschubotz  2019-01-18
+=====  ============  =========  ==========
+
+.. _82: https://github.com/gnosis/safe/issues/82
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+---------------------
+
+- There are some app features that we need to provide the user access to.
+    - Access Terms of Use
+    - Access Proviacy Policy
+    - Change password
+    - Enable/disable fingerprint
```

Comment:
what about other biometric features?

Reply:
good point, will make it more general and make sure to include them.

Reply:
So on iOS there is touch ID and face ID. They are  mutally exclusive, i.e. the phones with touch id don't have face ID and vice versa.

How about Android? Couldn't find too much on this online. Android Q is supposed to have a face ID equivalent. Is it also mutually exclusive? I think the Xiami in the office has both actually. So would we need 2 entries in the settings? 🤔 

cc @fmrsabino